src/components/consul/selectors/ImageSelector.js

```diff
@@ -103,6 +108,26 @@ export const ImageSelector = ({
       }
     }
 
+    if (selectorType === IMAGE_SELECTOR_TYPES.SUE) {
+      const { GPSLatitude, GPSLongitude } = imagesAttributes[index].exif || {};
```

Comment:
is safe that `imagesAttributes[index]` always exists? otherwise please `imagesAttributes?.[index]?.exif`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/b94d017584b4fcd1ad1be4b4f781619dbe741f1a